File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex14.c
#include <stdio.h>

main()
{
    printf("Reajuste Salarial\n\n");

    //vars
    float salarioI, salarioF;

    //Entrada
    printf("Insira o salario atual do funcionario: ");
    scanf("%f",&salarioI);

    //Processamento
    if ( salarioI <= 300 )
        salarioF = salarioI * 1.5;
    else if( salarioI <= 500 )
        salarioF = salarioI * 1.4;
    else if( salarioI <= 700)
        salarioF = salarioI * 1.3;
    else if ( salarioI <= 800 )
        salarioF = salarioI * 1.2;
    else if ( salarioI <= 100 )
        salarioF = salarioI * 1.1;
    else
        salarioF = salarioI * 1.05;

    //Saída
    printf("Novo salário: R$%.2f", salarioF);

}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex14.java
//Importando Scanner
import java.util.Scanner;

public class Ex14 {
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	System.out.println("Novo salário de um funcionário \n\n");
	
	//vars
	double salAtual, novoSal;
	
	//Entrada
	System.out.print("Insira o salário atual do funcionário: R$ ");
	salAtual = input.nextDouble();
	
	//Processamento
	if ( salAtual <= 300 )
		novoSal = salAtual * 1.5;
	else if ( salAtual <= 500 )
		novoSal = salAtual * 1.4;
	else if ( salAtual <= 700 )
		novoSal = salAtual * 1.3;
	else if ( salAtual <= 800 )
		novoSal = salAtual * 1.2;
	else if ( salAtual <= 1000 )
		novoSal = salAtual * 1.1;
	else
		novoSal = salAtual * 1.05;
	
	//Saída
	System.out.printf("Novo salário: R$%.2f", novoSal);
	
	}

}
